src/js/components/RadioButton/doc.js

```diff
@@ -18,6 +18,14 @@ export const doc = RadioButton => {
 
   DocumentedRadioButton.propTypes = {
     checked: PropTypes.bool.description('Same as React <input checked={} />'),
+    children: PropTypes.func.description(
```

Comment:
I think we should also add `children` prop to index.d.ts.

Reply:
Isn't it already there because of `React.ComponentClass`?

Reply:
I took a look at an example, and you are right. 
I suspect we should do a clean up on all the components that are defining `children` on their index.d.ts file. thanks for your comment.